### PR TITLE
BITMAKER-1997: Add send logs to kafka from entrypoint

### PR DESCRIPTION
**Pull request by Adriana618**
## Ticket
- https://tasks.bitmaker.dev/issues/1997

## Description
- Add Loghandler to send python logs to kafka.
- Add LogObserver to send scrapy logs to kafka.
- Add StdoutLogger to send stderr logs to kafka.

### CODE REVIEW

File: estela_scrapy/log.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        if msg:
+            msg = to_standar_str(msg[0])
+
+        failure = ev.get('failure', None)
+        if failure:
+            msg = failure.getTraceback()
+
+        why = ev.get('why', None)
+        if why:
+            msg = "%s\n%s" % (why, msg)
```
Comment (rodp63) [inline estela_scrapy/log.py]: Why are we using this notation?

File: estela_scrapy/log.py [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    root.setLevel(logging.NOTSET) # NOSET Make processing all messages if is set in root
+    
+    hdlr = LogHandler()
+    hdlr.setLevel(logging.INFO)
+    hdlr.setFormatter(logging.Formatter('[%(name)s] %(message)s'))
+    root.addHandler(hdlr)
+
+    # Silence commonly used noisy libraries
+    try:
+        import boto
```
Comment (rodp63) [inline estela_scrapy/log.py]: why do we need boto?

**Review by rodp63 — CHANGES_REQUESTED** [review]
Please apply the black style

File: estela_scrapy/log.py (hunk 1 above)
Comment (Adriana618) [reply estela_scrapy/log.py]: Changed.

File: estela_scrapy/log.py (hunk 2 above)
Comment (Adriana618) [reply estela_scrapy/log.py]: Removed.

**Review by rodp63 — APPROVED** [review]
Seems good

**Pull request merged**